src/pages/Product/components/Color/Color.js

```diff
@@ -1,18 +1,22 @@
 import React from "react";
 import "./Color.scss";
+import ColorButton from "../ColorButton/ColorButton";
+
+const Color = (props) => {
+
+  
 
-const Color = () => {
   return (
     <div className="color">
       <span className="colorText">
         색상 :
-        <div className="selected" />
-        white
+        <div className={`selected ${props.color}`}/>
+        {`${props.color}`}
       </span>
       <div className="colorHandler">
-        <button className="colorButton white" onClick={() => {}} />
-        <button className="colorButton red" onClick={() => {}} />
-        <button className="colorButton yellow" onClick={() => {}} />
+        <ColorButton className='colorButton white' color={'white'} setColor={props.setColor}/>
+        <ColorButton className='colorButton red' color={'red'} setColor={props.setColor} />
+        <ColorButton className='colorButton yellow' color={'yellow'} setColor={props.setColor} />
```

Comment:
```suggestion
        <ColorButton color='white' setColor={props.setColor}/>
        <ColorButton color='red' setColor={props.setColor} />
        <ColorButton color='yellow' setColor={props.setColor} />
```

ColorButton 컴포넌트에 넘겨준 className을 사용하지 않기도 하고
전달해준 color 값에 따라서 className이 변하기 때문에
위와 같이 변경할 수 있겠네요!

참고로 props를 전달해주는 방법 중 문자열을 데이터로 줄 때 중괄호로 씌우지 않아도 됩니다!